jekyll/_cci2/reusing-config.md

```diff
@@ -246,34 +246,48 @@ jobs:
 
 It is also possible to allow an orb to define the executor used by all of its commands. This allows users to execute the commands of that orb in the execution environment defined by the orb's author.
 
-### Example of Using an Executor Declared in config.yml in Multiple Jobs
+### Example of Using an Executor Declared in `config.yml` with Matrix Jobs.
 {:.no_toc}
 
-The following example declares and invokes an executor in two jobs that need to run in the same Docker image and working directory with a common set of environment variables. Each job has distinct steps, but runs in the same environment.
+The following example declares a Docker executor with a node image. The tag portion of the image string is parameterized with a `version` parameter. A `version` parameter is also included in our `test` job so that it can be passed through the job into the executor when the job is called from a workflow.
```

Comment:
```suggestion
The following example declares a Docker executor with a node image. The tag portion of the image string is parameterized with a `version` parameter. A `version` parameter is also included in the `test` job so that it can be passed through the job into the executor when the job is called from a workflow.
```